packages/core/faststore.config.default.js

```diff
@@ -29,7 +29,8 @@ module.exports = {
       symbol: '$',
     },
     locale: 'en-US',
-    channel: '{"salesChannel":"1","regionId":""}',
+    channel:
+      '{"salesChannel":"1","regionId":"","hasOnlyDefaultSalesChannel":"true"}',
```

Comment:
When the validateSession throws an error, the default channel is used. Because of this, the 'hasOnlyDefaultSalesChannel': 'true' property was added in faststore.config